testsuite/MDAnalysisTests/test_transformations.py

```diff
@@ -625,15 +623,34 @@ class TestRandomRotationMatrixCy(_RandomRotationMatrix):
 
 
 class _InverseMatrix(object):
+    def _check_inverse(self, size):
+        def makearr(size):
+            # Makes an array of shape (size,size) filled with
```

Comment:
I get the point of `makearr` but wouldn't this particular instance be a case for using `np.random.RandomState`? (I mean locally for this test, not to revert to my alternative). It'll be clearer and save many lines.